packages/luna/src/components/Lessons/UpcomingLessonsSummary.tsx

```diff
@@ -96,13 +120,34 @@ export const UpcomingLessonsSummary: React.FC<Props> = ({
                     >
                       {dayjs(lesson.start).format("dddd - D MMMM YYYY")}
                     </Typography>
-                    <Typography
-                      element="tiny-text"
-                      weight="regular"
-                      className="tw-text-natural-600"
-                    >
-                      {lesson.tutorName}
-                    </Typography>
+                    <div className="tw-flex tw-justify-between">
```

Comment:
Ensure that their is **at least `18px`** gap between the user name and lesson date.